### DISCUSSION TITLE
Error occurred while building a app in android studio

### DISCUSSION THREAD

**Discussion by ParthTagalpallewar** [organicmaps/organicmaps#3864] [Q&A]
This is something I got while running an application in android studio 

![error1](https://user-images.githubusercontent.com/55437315/201450139-fd280188-74ab-47ec-ada4-4e45dc5f7ccc.png)

![Screenshot (732)](https://user-images.githubusercontent.com/55437315/201450202-983095f6-fc6d-4bc6-9e93-cee880da8fc2.png)

**Comment by biodranik** [comment]
It looks like you did not follow the instructions and forgot to enable symlinks.

![image](https://user-images.githubusercontent.com/170263/201458298-e12029e1-b907-49c2-bb9e-bc784b325180.png)

**Reply by ParthTagalpallewar** [reply]
At first, I executed the above command ( git config --global core. symlinks true ) 
then I cloned the repository but still, it showed errors

Again I executed the command ( git config --global core. symlinks true ) 
Do I need to clone the repository again?

**Reply by biodranik** [reply]
Maybe no, but you should somehow recheckout it locally after enabling the flag. You may try to google it.

**Reply by rtsisyk** [reply]
> Do I need to clone the repository again?

Yes, please re-clone. It should work after.

**Comment by stevegdavis** [comment]
The above doesn't work for me, however if I use mklink on those 2 files then it does build with Windows 10 and Android Studio.

**Reply by biodranik** [reply]
@stevegdavis there should be a reason why your git for Windows doesn't support symlinks. I would suggest you google and fix that issue, because it may cause a lot of surprises later. Note that some of our submodules are also using symlinks.

**Comment by stevegdavis** [comment]
@biodranik found a suggestion on SO about enabling the symlinks during cloning:-
git clone --recurse-submodules **-c core.symlinks=true** https://github.com/organicmaps/organicmaps.git

This works fine for me, not sure why **git config --global core.symlinks true** doesn't.

**Reply by stevegdavis** [reply]
@ParthTagalpallewar - did you get this to work on Windows 10? and if so what params did you use. Thanks
